extensions/replication/utils/SetupReplication.js

```diff
@@ -417,14 +436,15 @@ class SetupReplication {
         });
     }
 
-    _enableReplication(roleArns, cb) {
+    _enableReplication(roleArns, siteName, cb) {
         const params = {
             Bucket: this._sourceBucket,
             ReplicationConfiguration: {
                 Role: roleArns,
                 Rules: [{
                     Destination: {
                         Bucket: `arn:aws:s3:::${this._targetBucket}`,
+                        StorageClass: siteName,
```

Comment:
I'm worried about the side effects here if `siteName` is undefined, depending on how the using code deals with own attributes that have undefined values. In doubt I would rather construct the `Destination` object separately only adding `StorageClass` if not undefined, but fine for me if you could validate that the replication config is correct with no `siteName`.

Reply:
I did check that there are no side effects, but it definitely doesn't hurt to construct the object separately—that can only be a good thing. Updated.